academy/3-my-own-chain/gas-meter.md

```diff
@@ -105,102 +117,251 @@ Additionally, making only a single call to `ConsumeGas` slightly saves computati
 
 </HighlightBox>
 
-## Integration tests
+## Unit tests
 
-Now you must add tests that confirm the gas consumption. However, it is not possible to differentiate the gas cost that BaseApp is incurring on your messages from the gas cost your module imposes on top of it. Also, you cannot distinguish via the descriptor [unless it panics](https://github.com/cosmos/cosmos-sdk/blob/v0.42.6/store/types/gas.go#L90-L101). Nevertheless, you can add a lame test:
+Now you must add tests that confirm the gas consumption. However, it is not possible to differentiate the gas cost that BaseApp is incurring on your messages from the gas cost your module imposes on top of it. Also, you cannot distinguish via the descriptor [unless it panics](https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/store/types/gas.go#L90-L101). Nevertheless, you can add a lame test like:
```

Comment:
Which version of the SDK are we actually using? I thought 0.46 🤔

Reply:
We used what Ignite provided unfortunately.